WcaOnRails/app/models/event.rb

```diff
@@ -22,6 +22,10 @@ def name_in(locale)
     I18n.t(id, scope: :events, locale: locale)
   end
 
+  def cellName
+    name
+  end
```

Comment:
Yeah! That was my first thought, but then I was under an impression that an exception would be thrown anyway - I forgot about that it's normal ActiveRecord attribute!